settings.php

```diff
@@ -76,10 +76,16 @@
 } else {
     $piHoleInterface = "unknown";
 }
-if (isset($setupVars["IPV4_ADDRESS"])) {
-    $piHoleIPv4 = $setupVars["IPV4_ADDRESS"];
+
+// get the gateway address for the default route on the Pi-hole interface
+if ($piHoleInterface !== "unknown") {
+$IPv4GW=shell_exec("ip -4 route | grep default | grep '${piHoleInterface}' | cut -d ' ' -f 3");
 } else {
-    $piHoleIPv4 = "unknown";
+    $IPv4GW = "unknown";
+}
```

Comment:
Maybe this `else` is not needed, since the next line will test `$IPv4GW`.

Reply:
True. I removed it.